src/component/myForm/CreatePlaneForm.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import {
+  ErrorMessage, Field, Form, Formik,
+} from 'formik';
+import { createPlane } from '../../redux/slices/planesSlice';
+import { CreatePlaneSchema } from '../lib/schema';
+import NavBar from '../nav/navBar';
+import styles from './form.module.scss';
+
+const CreatePlaneForm = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <NavBar />
+      <section
+        className={styles.mainContainer}
+        style={{ backgroundColor: 'var(--color1)' }}
+      >
+        <h2 style={{ color: 'white', fontSize: '3rem' }}>Provide plane details</h2>
```

Comment:
- [ ] If you can use classes for styling here, it would be nice, since it's not a good practice to use the `style` attribute.